src/Components/PhotoUpload/Utils/uploadFileToS3.tsx

```diff
@@ -23,16 +30,27 @@ export const uploadFileToS3 = (
     const bucket = asset.policyDocument.conditions.bucket
     const uploadURL = `https://${bucket}.s3.amazonaws.com`
 
+    const isExternalPhoto = photo.externalUrl
+
+    let file
+
+    try {
+      file = isExternalPhoto ? await externalPhotoToFile(photo) : photo.file
+    } catch (error) {
+      reject(new Error("Initializing artwork photo failed."))
+      return
+    }
+
     const data = {
-      acl: acl,
-      "Content-Type": photo.file.type,
+      acl,
+      "Content-Type": photo.file.type || "image/jpg",
```

Comment:
**thought:** I am wondering if it would be possible to always have the file type here to avoid the `|| image/jpg` here

Reply:
You're right, the file type should always be set. I've removed the `|| "image/jpg"`.